src/nnfusion/frontend/onnx_import/op/const_of_shape.hpp

```diff
@@ -44,19 +44,29 @@ namespace nnfusion
                     NNFUSION_CHECK(GetValueFromNGraphOp(input.gnode, &output_shape));
 
                     Node node(node_proto);
-                    auto value = node.get_attribute_value<Tensor>("value");
-                    NNFUSION_CHECK(nnfusion::shape_size(value.get_shape()) == 1);
-                    auto const_op =
-                        make_constant_op(value.get_ng_type(),
-                                         Shape(std::begin(output_shape), std::end(output_shape)),
-                                         value);
+                    std::shared_ptr<nnfusion::op::Constant> const_op;
+                    NNFUSION_LOG(INFO) << Shape(std::begin(output_shape), std::end(output_shape));
+                    try
+                    {
+                        auto value = node.get_attribute_value<Tensor>("value");
```

Comment:
try catch is for? Using node.has_attribute is enough?

Reply:
thanks for pointing it out, node.has_attribute() is better

Reply:
> try catch is for? Using node.has_attribute is enough?

thanks for pointing it out, node.has_attribute() is better